server/data/prisonApiClient.ts

```diff
@@ -1,11 +1,62 @@
 import RestClient from './restClient'
 import { CaseLoad } from '../interfaces/caseLoad'
+import { RedisClient } from './redisClient'
+
+const API_COOL_OFF_TIME = 300
```

Comment:
The cool off time is 5 mins here, do we want to shorten this a little? happy either way just want to see if this is neccesary.

Reply:
it is an arbitrary value really, just to give it time to recover. Happy to go with whatever people think. However, the worst that can happen is that the caseload switcher doesn't display while the prison api is not being called; it doesn't error